src/tests/test_error_messages.py

```diff
@@ -0,0 +1,177 @@
+import pytest
+
+from sqlalchemy.exc import OperationalError
+from IPython.core.error import UsageError
+
+
+@pytest.fixture
+def query_no_suggestion():
+    return (
+        "sql",
+        "",
+        """
+        sqlite://
+        SELECT FROM author;
+        """,
+    )
+
+
+@pytest.fixture
+def query_description():
```

Comment:
For me, the naming convention `query_coloumn_with_syntax_error` makes more sense to me.
But I won't block this, up to you.